synnax/pkg/ranger/action.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+package ranger
+
+import "github.com/synnaxlabs/synnax/pkg/access/action"
+
+const (
+	SetKVAction        action.Action = "set_key_value"
```

Comment:
Did these actions exists previously? Regardless, I don't think they should exists. Why can't the `set_key_value` be defined as a `create` on a `ranger.KVPair` object? We're defining a bunch of unnecessary actions.

Reply:
For the KVpair actions, I think it makes more sense to define it as an `update` on a `range`, right? The KV pairs are not being tracked in the ontology. Regardless, I can get rid of these. I think the aliases should still be kept, but I can consolidate them into one `edit_alias` action.

Reply:
I'm fine with using `update` on `range`. We should get rid of these